chapter/tail-calls.scrbl

```diff
@@ -836,8 +835,8 @@ metadata.
 @defproc[(uncover-locals (p asm-pred-lang-v5?))
           asm-pred-lang-v5/locals?]{
 Compiles @tech{Asm-pred-lang v5} to @tech{Asm-pred-lang v5/locals}, analysing
-which @ch2-tech{abstract locations} are used in each block, and each block and the
-module with the set of variables in an @racket[info?] fields.
+which @ch2-tech{abstract locations} are used in each block, and updating each
+block and the module with the set of variables in an @racket[info?] fields.
```

Comment:
I wasn't sure about this one but I think it has to do with updating the info fields in each of the define blocks and also the main one?